Mathlib/Combinatorics/Additive/FreimanHom.lean

```diff
@@ -246,12 +246,18 @@ lemma isMulFreimanIso_empty : IsMulFreimanIso n (∅ : Set α) (∅ : Set β) f
   mapsTo := hfAB
   map_prod_eq_map_prod s t _ _ _ _ h := by rw [← map_multiset_prod, h, map_multiset_prod]
 
-@[to_additive] lemma MulEquivClass.isMulFreimanIso [EquivLike F α β] [MulEquivClass F α β] (f : F)
+@[to_additive] lemma MulHomClass.isMulFreimanIso [EquivLike F α β] [MulHomClass F α β] (f : F)
     (hfAB : BijOn f A B) : IsMulFreimanIso n A B f where
   bijOn := hfAB
   map_prod_eq_map_prod s t _ _ _ _ := by
     rw [← map_multiset_prod, ← map_multiset_prod, EquivLike.apply_eq_iff_eq]
 
+@[deprecated (since := "2025-08-21")]
+alias MulEquivClass.isMulFreimanIso := MulHomClass.isMulFreimanIso
+
+@[deprecated (since := "2025-08-21")]
+alias AddEquivClass.isAddFreimanIso := AddHomClass.isAddFreimanIso
```

Comment:
I feel that if a decl needs to assume EquivLike then we should keep using Equiv in the name ... although in this case `Iso` conveys the meaning of `Equiv`.